config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:show, :new, :create]
+  resources :sessions, only: [:new, :create, :destroy]
+  resources :events, only: [:index, :show, :new, :create]
+  resources :atendences
```

Comment:
- [ ] Since you only have a create method for this controller you should specify the resources only for the create method i.e `resources :atendences, only: [:create]`